app/code/Magento/Checkout/view/frontend/web/js/sidebar.js

```diff
@@ -261,6 +261,9 @@ define([
                 $(document).trigger('ajax:removeFromCart', {
                     productIds: [productData['product_id']]
                 });
+                if (window.location.href === this.shoppingCartUrl) {
+                    location.reload();
```

Comment:
Would be great to replace to 
```suggestion
                    window.location.reload();
```